app/views/users/index.html.erb

```diff
@@ -19,7 +19,7 @@
   <tbody>
     <% @users.each do |user| %>
       <tr>
-        <td class="display-two-pane ellipsis"><%= avatar_image_tag user, :class => "avatar" %> <%=link_to_if_authorized user.login, hash_for_edit_user_path(:id => user).merge(:auth_object => user, :authorizer => authorizer) %></td>
+        <td class="display-two-pane ellipsis"><%= user.mail %> <%=link_to_if_authorized user.login, hash_for_edit_user_path(:id => user).merge(:auth_object => user, :authorizer => authorizer) %></td>
```

Comment:
The email here looks weird + it's a duplicate info, as we already have the email column.

![Uploading avatar-4.png…]()

Reply:
And since the ldap avatar support should IMO stay, I think we can leave this untouched for now.